require.js

```diff
@@ -69,13 +67,11 @@
     if (loaderPath == null) loaderPath = '/';
     path = unalias(name, loaderPath);
 
+    if (!has.call(modules, path)) path = aliases[path];
```

Comment:
These should not conflict with the component aliases. The component aliases all end with `.js`. The aliases we are adding are for the file without an extension or the directory if it is an index.js file.